.changeset/heavy-countries-drop.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
We don’t have to add changeset in this scenario, as it is not supposed to create version bump.

Reply:
Done